### DISCUSSION TITLE
ESET antivirus found GenKryptik malware in copilot

### DISCUSSION THREAD

**Discussion by bbtrash** [community/community#20247] [Projects and Issues]
Hello, 

ESET antivirus detects Win64/GenKryptik.FXIE on location : ... AppData\Roaming\JetBrains\PhpStorm2021.2\plugins\github-copilot-intellij\copilot-agent\bin\copilot-agent-win.exe.

File was deleted, Copilot is now offline. Plugin reinstall not working.

Do you have experience with the similar problem ? 

Thanks

**Comment by radar-blip** [comment]
Similar issue here: https://github.com/orgs/github-community/discussions/20268
